tests/integration/session_integration/test.rfi

```diff
@@ -4,3 +4,6 @@ Session:
             Default: 2048
             Global:
                 Message: 1024
+            Local:
+                Start:
+                  M: 1024
```

Comment:
I think the test could be improved by using a different size for `Message` and `M`.

Reply:
fixed in 3c684cf25a88b85b6cf8664ddc5096cd6e4bc2a0